spec/chunk-processor.spec.js

```diff
@@ -3,60 +3,73 @@ var ChunkProcessor = require('../src/Resources/chunk-processor');
 describe("chunk-processor", function () {
     it("protocolReceiveBytes", function () {
         var chunkProcessor = new ChunkProcessor();
-        expect(chunkProcessor.protocolReceiveBytes).toEqual(7);
+        expect(chunkProcessor.numberOfBytesToAsk()).toEqual(7);
     });
 
     it("processChunk in large bits", function () {
         var chunkProcessor = new ChunkProcessor();
         // Send the header, we need 12 bytes of file contents afterwards
         var chunk = Buffer.alloc(7);
-        chunk.writeUInt32LE(12);
-        chunk.write('ABC', 4);
+        chunk.write('ABC');
+        chunk.writeUInt32LE(9, 3);
```

Comment:
did you switch order intentionally?

Reply:
Yeah, we switched the order in the protocol, so now we first send the type, then the file length.